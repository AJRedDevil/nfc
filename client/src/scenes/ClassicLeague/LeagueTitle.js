// npm packages
import React from 'react';
import PropTypes from 'prop-types';

// our packages
import {DateHelper} from '../../utils';

const LeagueTitle = ({leagueName, creationDate}) => (
  <div>
    <h1 className="title">{leagueName}</h1>
    <h2 className="subtitle">
      Created : {DateHelper.ISOToDateString(creationDate)}
    </h2>
  </div>
);
LeagueTitle.propTypes = {
  leagueName: PropTypes.string.isRequired,
  creationDate: PropTypes.string.isRequired,
};

export default LeagueTitle;
