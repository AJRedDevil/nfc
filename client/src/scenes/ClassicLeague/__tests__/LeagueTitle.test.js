// npm packages
import React from 'react';
import renderer from 'react-test-renderer';

// our packages
import LeagueTitle from '../LeagueTitle';

test('# LeagueTitle', () => {
  const wrapper = renderer
    .create(
      <LeagueTitle
        leagueName="TestLeague"
        creationDate="2018-02-21T00:41:36.214Z"
      />
    )
    .toJSON();
  expect(wrapper).toMatchSnapshot();
});
