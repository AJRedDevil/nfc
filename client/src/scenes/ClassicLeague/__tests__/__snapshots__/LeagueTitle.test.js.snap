// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# LeagueTitle 1`] = `
<div>
  <h1
    className="title"
  >
    TestLeague
  </h1>
  <h2
    className="subtitle"
  >
    Created : 
    Wed Feb 21 2018
  </h2>
</div>
`;
